srv/components/provisioner/files/cluster_sls.template

```diff
@@ -38,7 +40,7 @@ cluster:
   srvnode-{{ (num + 1) }}:
     rack_id:
     site_id:
-    storage_set_id: storage-set-1
+    storage_set_id: storage-set-{{  (num//3) + 1 }}
```

Comment:
please add spaces around binary operator here as well

Reply:
Also remove the extra space before: `(num//3)`

Reply:
I've updated the code in a new commit